src/Test.Utilities.Sarif/TestUtilitiesExtensions.cs

```diff
@@ -24,11 +24,9 @@ public static void ValidateCommandExecution(this IAnalysisContext context, int r
 
             // For application exist exceptions, e.g., an unhandled exception in a rule,
             // the inner exception has the most useful details.
-            context.RuntimeExceptions?[0].InnerException.Should().BeNull();
-            context.RuntimeExceptions?[0].Should().BeNull();
-
+            context.RuntimeExceptions?[0].InnerException?.ToString().Should().BeNull();
```

Comment:
>ToString [](http://example.com/codeflow?start=58&length=8)

These are subtle changes that publish more actionable data to the test logs when failures occur. Now we'll get all of ex.ToString() (which is very rich and useful) rather than a limited report that's constrained to the exception type.